cabot/cabotapp/models.py

```diff
@@ -98,6 +99,12 @@ class Meta:
         blank=True,
         help_text='Users who should receive alerts.',
     )
+    schedules = models.ManyToManyField(
+        'Schedule',
+        blank=True,
+        null=True,
+        help_text='Oncall schedule to be alerted.'
+    )
```

Comment:
probably, we should remove `instance` support from our version of cabot (to be renamed).
Will avoid confusion.